tools/issue-token.sh

```diff
@@ -32,6 +32,6 @@ echo "email address is: $email"
 read -p "Enter Your Password: "  password
 read -p "Enter Your Name: "  name
 
-echo "{\"id\":\"${id}\",\"email\":\"${email}\",\"password\":\"${password}\",\"name\":\"${name}\"}" > user.json
+echo "{\"email\":\"${email}\",\"password\":\"${password}\",\"name\":\"${name}\"}" > user.json
```

Comment:
Storing the password seems unnecessary, and insecure (it seems to getting stored in plaintext)

Reply:
Good point. Thanks.